### DISCUSSION TITLE
Glitches in some animation frames - anyone experience this?

### DISCUSSION THREAD

**Discussion by evanbunnage** [BradyAJohnston/MolecularNodes#178] [Q&A]
I'm creating an animation using a single protein. Weirdly, I notice that about a quarter of my rendered frames seem to lose some of the atoms. Does anyone else run into this? This scene is pretty simple: one PDB model, a couple lights and a background. Is this a blender thing? I've uploaded a little clip that shows the flickering, about ~10 of the 20 frames have the issue. Its hard to Google this issue, apologies if its been asked.

https://user-images.githubusercontent.com/100053509/225147309-f27d55e0-54d5-4431-810a-f6c8b70be357.mov

**Comment by BradyAJohnston** [comment]
Are you able to share your node tree / blend file?  I can have a look at it. I can remember I've had flickering in the past but I don't remember what caused it but it's not a problem anymore for me (might be file specific)

**Reply by evanbunnage** [reply]
I appreciate that! I overrwrote the file messing around with settings trying to fix this 🤦‍♂️ so it might not reproduce the bug.

[This post](https://blender.stackexchange.com/questions/182227/weird-flickering-in-my-model-when-i-rotate-the-camera-and-even-if-its-fixed) and a few others recommended changing the "Clip start" setting in the camera and it seemed to help when I did a quick render on my laptop. I'm running a full size render on my CUDA device now, I'll report back if it fixes it or not (with a blend file). 15 frames in and its okay 🤞

**Reply by BradyAJohnston** [reply]
That's good to hear! I remember it being also quite inconsistent, and may have been CPU / GPU different. Keep me posted!

**Reply by evanbunnage** [reply]
Okay, 350 frames later I think its safe to call this solved. I did run into this a few months ago too so I'm surprised its not more common!

I ruled out the render device (happened on CPU, CUDA and CPU+CUDA).

For anyone in the future, go to your camera then change the Clip Start to be a larger number, I chose 1m (see screenshot). I'm not sure the exact cause, but it sounds like a large difference in the clip start and end can cause some floating point rounding errors in the render which explains the randomness.

Here's my fix:

<img width="468" alt="Screenshot 2023-03-14 at 8 50 04 PM" src="https://user-images.githubusercontent.com/100053509/225201620-f8434356-38a6-4285-b467-fe17ef7d1da4.png">

**Reply by BradyAJohnston** [reply]
Weird that it was only happening to a specific subset of the protein, and that subset was getting the error over and over again. Thanks for the info!
